src/pages/Dictionary/components/DisplayMode.jsx

```diff
@@ -1,24 +1,124 @@
 import { Button } from '@nextui-org/react';
 import { LikeFilled, DislikeFilled } from '@ant-design/icons';
+import {
+  incrementLikes,
+  incrementDislikes,
+  decrementDislikes,
+  decrementLikes
```

Comment:
these functions can be simplified into 2 functions. pass true/false value to the function.
handleFirebaseLikes(entryid, true) and handleFirebaseDislikes(entryid, true)